### ISSUE TITLE
Errors in starting up the application

### ISSUE THREAD

**Issue by ziyannchen** [netless-io/flat#1480]
export * as ipc from "./ipc";
^^^^^^
SyntaxError: Unexpected token 'export'
    at wrapSafe (internal/modules/cjs/loader.js:986:16)
    at Module._compile (internal/modules/cjs/loader.js:1034:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1108:10)
    at Module.load (internal/modules/cjs/loader.js:935:32)
    at Module._load (internal/modules/cjs/loader.js:776:14)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12913)
    at Module.require (internal/modules/cjs/loader.js:959:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.flat-types (D:\WorkSpace\Project\flat\desktop\main-app\dist\main.js:1288:18)
    at __webpack_require__ (D:\WorkSpace\Project\flat\desktop\main-app\dist\main.js:1382:42)

Hello, thanks for your nice open-source flat application.
I try to run a desktop version flat, but I just run into this error without any modification to the original project.
Is it an ES related problem? How can I solve that?

**Comment by crimx** [comment]
Hi! Did you follow the [Installation Guide](https://github.com/netless-io/flat#installation)? In which step did the error occur?

**Comment by ziyannchen** [comment]
Hello! Thx for your prompt reply. I've tried again yesterday, and it just worked fine this time.
I have no idea what went wrong last time.
Anyway, thanks again for your reply and maintenance of this nice project.

**Comment by ziyannchen** [comment]
Hi, I just found out that there is another problem.
It seems be about the devtools of electron, which makes me fail to start the flat electron.
`[14432:0421/104053.885:ERROR:CONSOLE(1)] "Extension server error: Object not found: <top>", source: devtools://devtools/bundled/extensions/extensions.js (1)`
I went through the maintenance history, there seems be a record about this.
However, I downloaded the latest version and also tried different project versions to start the electron, but they didn't work.
The electron just didn't load anything but an empty page.
Btw, **start:web** can be successfully implemented.
